app/models/manageiq/providers/cisco_intersight/physical_infra_manager/ems_refresh_workflow.rb

```diff
@@ -0,0 +1,43 @@
+class ManageIQ::Providers::CiscoIntersight::PhysicalInfraManager::EmsRefreshWorkflow < ManageIQ::Providers::EmsRefreshWorkflow
+  def run_native_op
+    queue_signal(:poll_native_task)
+  end
+  alias start run_native_op
+
+  def poll_native_task
+    wf_api = ManageIQ::Providers::CiscoIntersight::PhysicalInfraManager.first.connect(:service=>'WorkflowApi')
```

Comment:
Same comment here on using `ManageIQ::Providers::CiscoIntersight::PhysicalInfraManager.first` this should use the ems_id that you passed in, or `target.ext_management_system`